<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-24T21:49:22 IST" name="Default suite" finished-at="2024-04-24T21:49:38 IST" duration-ms="15938">
    <groups>
    </groups>
    <test started-at="2024-04-24T21:49:22 IST" name="Default test" finished-at="2024-04-24T21:49:38 IST" duration-ms="15938">
      <class name="com.QA.DataDrivenScripts.SailpointDDTLogin">
        <test-method is-config="true" signature="generateReports()[pri:0, instance:com.QA.DataDrivenScripts.SailpointDDTLogin@2187fff7]" started-at="2024-04-24T21:49:22 IST" name="generateReports" finished-at="2024-04-24T21:49:22 IST" duration-ms="128" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateReports -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.QA.DataDrivenScripts.SailpointDDTLogin@2187fff7]" started-at="2024-04-24T21:49:22 IST" name="setUp" finished-at="2024-04-24T21:49:25 IST" duration-ms="2862" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="Launch_Application_DDT()[pri:1, instance:com.QA.DataDrivenScripts.SailpointDDTLogin@2187fff7]" started-at="2024-04-24T21:49:25 IST" name="Launch_Application_DDT" finished-at="2024-04-24T21:49:37 IST" duration-ms="12641" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Launch_Application_DDT -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.QA.DataDrivenScripts.SailpointDDTLogin@2187fff7]" started-at="2024-04-24T21:49:37 IST" name="tearDown" finished-at="2024-04-24T21:49:38 IST" duration-ms="381" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="closeReport()[pri:0, instance:com.QA.DataDrivenScripts.SailpointDDTLogin@2187fff7]" started-at="2024-04-24T21:49:38 IST" name="closeReport" finished-at="2024-04-24T21:49:39 IST" duration-ms="943" status="FAIL">
          <exception class="java.io.IOException">
            <message>
              <![CDATA[Failed to open file:/C:/Users/na21279/eclipse-workspace/SailPoint_Automation/./reports/Reports__24-04-2024%2021-49-22/report_24-04-2024%2021-49-22.html. Error message: The system cannot find the file specified.
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.IOException: Failed to open file:/C:/Users/na21279/eclipse-workspace/SailPoint_Automation/./reports/Reports__24-04-2024%2021-49-22/report_24-04-2024%2021-49-22.html. Error message: The system cannot find the file specified.

at java.desktop/sun.awt.windows.WDesktopPeer.ShellExecute(WDesktopPeer.java:115)
at java.desktop/sun.awt.windows.WDesktopPeer.browse(WDesktopPeer.java:101)
at java.desktop/java.awt.Desktop.browse(Desktop.java:533)
at com.QA.Application.TestBase.TestBase.closeReport(TestBase.java:87)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:386)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.IOException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeReport -->
      </class> <!-- com.QA.DataDrivenScripts.SailpointDDTLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
